### PR TITLE
fix(#1998): improves error message when quarkus:dev fails on empty project

### LINKED ISSUE #1998: quarkus:dev fails on empty project
Creates an empty project, without any sources or resources and run:

```
mvn compile quarkus:dev
...
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:0.13.1:dev (default-cli) on project async-demo: The project has no output yet, run `mvn compile quarkus:dev`. -> [Help 1]
```

Either the error messages must be updated to mention that the project must at least contain 1 file, or it should run and waits until we have content.

### PR DESCRIPTION
**Pull request by machi1990**
Fixes #1998

### CODE REVIEW

File: devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java [hunk 1]
```diff
@@ -186,7 +186,8 @@ public void execute() throws MojoFailureException, MojoExecutionException {
         }
 
         if (!buildDir.isDirectory() || !new File(buildDir, "classes").isDirectory()) {
-            throw new MojoFailureException("The project has no output yet, run `mvn compile quarkus:dev`.");
+            throw new MojoFailureException("The project " + project.getName()
+                    + " has no output yet. Make sure it contains at least a source or resource file and then run `mvn compile quarkus:dev`.");
```
Comment (gsmet) [inline devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java]: I think, in this case, I would use `at least one` and not `at least a`.

Could you fix it?

**Review by gsmet — CHANGES_REQUESTED** [review]
Added a minor comment.

File: devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java (hunk 1 above)
Comment (machi1990) [reply devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java]: Done.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**